protocol/messenger_mailserver.go

```diff
@@ -937,61 +948,8 @@ func (m *Messenger) ConnectionChanged(state connection.State) {
 	}
 
 	if m.connectionState.Offline && !state.Offline {
-		err := m.sender.StartDatasync()
-		if err != nil {
-			m.logger.Error("failed to start datasync", zap.Error(err))
-		}
+		m.sender.StartDatasync()
```

Comment:
why removing the error checking, had no impact on the functioning but it would report an error?

Reply:
It was removed by mistake